src/components/HowRarimoWorksSection/HowRarimoWorksSection.jsx

```diff
@@ -112,46 +113,215 @@ const HowRarimoWorksSection = () => {
     };
   }, [containerHeight]);
 
+  useEffect(() => {
+    if (isDesktop) return;
+    const paramsMobile = {
```

Comment:
no need to create this object on each useEffect execution

Reply:
fixed